src/util/ReturnsCalculations.ts

```diff
@@ -26,14 +25,16 @@ export function calculateReturns(poolReturns: PoolReturns, token0Returns: TokenR
 
     // Copy to avoid mutating the actual data
     const poolReturnsCopy = [...poolReturns];
```

Comment:
I don't think we need this copy anymore, since `intersectionOfReturns` will do so implicitly